src/calculator.m

```diff
@@ -0,0 +1,22 @@
+classdef Calculator
+    methods(Static)
+        function result = add(a, b)
```

Comment:
Static methods can lead to tight coupling. Consider using instance methods or a different design pattern to improve modularity.